internal/jimm/jimm.go

```diff
@@ -131,6 +131,10 @@ func (j *JIMM) AuthorizationClient() *openfga.OFGAClient {
 	return j.OpenFGAClient
 }
 
+func (j *JIMM) GetRoleManager() RoleManager {
+	return j.RoleManager
```

Comment:
the same goes for OpenFGAClient, Pubsubhub, database, etc etc.

I think you suggested the newJIMM constructor and i fully agree.
Otherwise we need to add some "panic" in these getters.
I would leave as it is now

Reply:
card created.